components/SeasonTabs.vue

```diff
@@ -15,19 +15,12 @@
           :infinite="false"
           :disable-arrows-on-edges="true"
           :bullets="false"
+          :breakpoints="breakpoints"
           :dragging-distance="200"
         >
           <vueper-slide v-for="episode in season.Episodes" :key="episode.Id">
             <template v-slot:content>
-              <v-card>
-                <v-img :src="getImageLink(episode.Id, 'primary')"></v-img>
-                <v-card-subtitle>
-                  {{
-                    $t('episodeNumber', { episodeNumber: episode.IndexNumber })
-                  }}
-                </v-card-subtitle>
-                <v-card-title>{{ episode.Name }}</v-card-title>
-              </v-card>
+              <card :item="episode" :options="{ subtitle: 'episodeNumber' }" />
```

Comment:
Looks like you forgot to change it :smiley: 

```suggestion
              <card :item="episode" episode />
```

Reply:
Oops :p